app/services/api/axios.ts

```diff
@@ -26,163 +24,163 @@ export const axiosInstance = axios.create({
-          type: "976a2eee2ab173440affe93d0a20bf4d",
-          day: "ed2cfa99e27dce5d1a425a419f170eb5",
-          "break-party-description":
-            "Check-in for attendees with a workshop ticket begins at 6:00 am.",
-          name: "Check-in & Registration",
-          slug: "morning-break-day-2",
-          "speaker-2": "637c0437faeaa766e0b15eac",
-          // random data
-          _archived: false,
-          _draft: false,
-          "updated-on": "2020-02-26T18:54:55.439Z",
-          "updated-by": "Person_5ca77d28397726353bf26f69",
-          "created-on": "2020-02-22T01:00:03.417Z",
-          "created-by": "Person_5ca77d28397726353bf26f69",
-          "published-on": "2022-11-21T23:24:43.625Z",
-          "published-by": "Person_57fc009927a6922a35903a0b",
-          _cid: "6362b046e69e80096f361102",
-          _id: "6362b046e69e808763361159",
-        },
-        {
-          "break-card-color": "hsla(244.28571428571433, 91.30%, 90.98%, 1.00)",
-          "day-time": "2023-05-19",
-          type: "976a2eee2ab173440affe93d0a20bf4d",
-          day: "ed2cfa99e27dce5d1a425a419f170eb5",
-          name: "Leveling up on the new architecture",
-          "break-party-description": "Gant Laborde",
-          slug: "registration-day-2",
-          "speaker-2": "637c0437faeaa766e0b15eac",
-          // random data
-          _archived: false,
-          _draft: false,
-          "updated-on": "2020-02-22T00:59:20.432Z",
-          "updated-by": "Person_5ca77d28397726353bf26f69",
-          "created-on": "2020-02-22T00:59:20.432Z",
-          "created-by": "Person_5ca77d28397726353bf26f69",
-          "published-on": "2022-11-21T23:24:43.625Z",
-          "published-by": "Person_57fc009927a6922a35903a0b",
-          _cid: "6362b046e69e80096f361102",
-          _id: "6362b046e69e80dde2361178",
-        },
-        {
-          "break-card-color": "hsla(244.28571428571433, 91.30%, 90.98%, 1.00)",
-          "day-time": "2023-05-19",
-          type: "976a2eee2ab173440affe93d0a20bf4d",
-          day: "ed2cfa99e27dce5d1a425a419f170eb5",
-          name: "React Native case study: from an idea to market",
-          "break-party-description": "Ferran Negre Pizarro",
-          slug: "registration-day-2",
-          "speaker-2": "637c0437faeaa766e0b15eac",
-          // random data
-          _archived: false,
-          _draft: false,
-          "updated-on": "2020-02-22T00:59:20.432Z",
-          "updated-by": "Person_5ca77d28397726353bf26f69",
-          "created-on": "2020-02-22T00:59:20.432Z",
-          "created-by": "Person_5ca77d28397726353bf26f69",
-          "published-on": "2022-11-21T23:24:43.625Z",
-          "published-by": "Person_57fc009927a6922a35903a0b",
-          _cid: "6362b046e69e80096f361102",
-          _id: "6362b046e69e80dde2361178",
-        },
-      ]
-      response.data.items = mockScheduleData
-    }
+    // if (response.config.url === `/collections/${SCHEDULE.collectionId}/items`) {
```

Comment:
Not sure if I should delete this mock data or not, just in case

Reply:
If these mocked ones have all been replaced with the live data I'd look to remove it. We can always grab them back from the git history in the file if something needs to be tested that way.

Reply:
Agreed!